### DISCUSSION TITLE
Github pages much slower than local?!

### DISCUSSION THREAD

**Discussion by Illuminarchie66** [Leaflet/Leaflet#9325] [Q&A]
Hiya !

Cutting to the chase, I am trying to make an interactive map, of an svg that I have drawn. I really love the markers and the interactive elements of leaflet, so I assumed this was the best method. The relevant code is below.
```
<object type="image/svg+xml" data="mapit4.svg" id="svgObject"></object>
...
<script>
       // Initialize the map
        var map = L.map('map', {
            zoomDelta: 0.5,
            renderer: L.canvas(),
            doubleClickZoom: false
        }).setView([0, 0], 5); // Set initial view (lat, lng), zoom level

        //Create an SVG element
        var svgElementBounds = [[-720, -480], [720, 480]]; // Set bounds as [[lat1, lng1], [lat2, lng2]] or [[y1, x1], [y2, x2]]
        var svgObject = document.getElementById('svgObject');
        // Add the SVG overlay to the map
        L.svgOverlay(svgObject, svgElementBounds).addTo(map);
        

        map.setMaxBounds([
            [-720, -480],
            [720, 480]
        ]);

        map.setMaxZoom(10); // Set maximum zoom level to 15
        map.setMinZoom(2); // Set minimum zoom level to 5
</script>
```

When I run this locally, its totally fine! Its quick and snappy and exactly what I wanted. However when running it on Github-Pages, it is very slow and stuttery with how it all moves. I can't understand why this is happening! 

Any help would be greatly appreciated! 

You can also experience it as well 
https://illuminarchie66.github.io/Mario-Map/

Please let me know if I'm being a total idiot

**Comment by mourner** [comment]
Can you say more about the choppiness? Which browsers did you test, on which operating systems? Is performance bad on all of them, or only particular ones? I just tried in Chrome / Mac M1 and your map seems to work fine.

**Reply by Illuminarchie66** [reply]
Thank you so much for replying ! That is very interesting to know
The choppiness and lagginess that I am experiencing is demonstrated in the videos below:
Github run (choppy)
https://github.com/Leaflet/Leaflet/assets/54813067/290d0c33-48ae-46fe-8357-5d3478ad7035

Local run (desired)
https://github.com/Leaflet/Leaflet/assets/54813067/d8dcdf80-21f3-470a-af02-0f7100b8d53f

I tried the map on Chrome and Microsoft Edge on both of their latest versions on my Windows device.
I just tried it on my phone on Safari and it ran significantly better than Chrome and Microsoft Edge.
